src/gui/torrentcontentmodelitem.h

```diff
@@ -35,47 +35,51 @@
 #include <QVariant>
 
 namespace prio {
-enum FilePriority {IGNORED=0, NORMAL=1, HIGH=6, MAXIMUM=7, MIXED=-1};
+    enum FilePriority {IGNORED=0, NORMAL=1, HIGH=6, MAXIMUM=7, MIXED=-1};
 }
 
 class TorrentContentModelFolder;
 
-class TorrentContentModelItem {
+class TorrentContentModelItem
+{
 public:
-  enum TreeItemColumns {COL_NAME, COL_SIZE, COL_PROGRESS, COL_PRIO, COL_REMAINING, NB_COL};
-  enum ItemType { FileType, FolderType };
+    enum TreeItemColumns {COL_NAME, COL_SIZE, COL_PROGRESS, COL_PRIO, COL_REMAINING, COL_AVAILABILITY, NB_COL};
```

Comment:
I don't want to be an author from the Git point of view for the whole files which I touch. Git can ignore whitespace changes and thus I fixed indentation, but not more. What is the point in changing code which is not related to a functional changes of a given commit? This just spoils git history and makes history navigation a nightmare.